frontend/src/reviews/ReviewRoleDisplay.tsx

```diff
@@ -0,0 +1,32 @@
+import { Typography } from '@mui/material'
+
+import { useGetModelRoles } from '../../actions/model'
+import { ReviewRequestInterface } from '../../types/interfaces'
+import { getRoleDisplay } from '../../utils/beta/roles'
+import Loading from '../common/Loading'
+import MessageAlert from '../MessageAlert'
+
+type DisplayRoleProps = {
```

Comment:
This should match the name of the component, so rename to `ReviewRoleDisplayProps`